### DISCUSSION TITLE
新版职教云mooc中断问题

### DISCUSSION THREAD

**Discussion by yangfeng20** [11273/mooc-work-answer#88] [General]
运行新版的mooc刷课，运行一段时间之后就会被拦截，这个是因为太频繁导致的吗，如果是设置延时可以解决这个问题吗？

**Comment by 11273** [comment]
估计不能，需要alicfw过防火墙就可以，最新代码需要手动获取到alicfw会稳定一点

**Reply by yangfeng20** [reply]
我试了使用浏览器的cookie，可以稍微避免这个问题。

**Reply by yangfeng20** [reply]
https://github.com/yangfeng20/mooc-work-answer/blob/header_auth/NewMoocMain/init_mooc.py
